app/controllers/signup.js

```diff
@@ -1,194 +1,219 @@
-import Controller from '@ember/controller';
-import { tracked } from '@glimmer/tracking';
-import { action, set } from '@ember/object';
+import Controller from "@ember/controller";
+import { tracked } from "@glimmer/tracking";
+import { action, set } from "@ember/object";
+import HistoryLocation from "@ember/routing/history-location";
+
+// const BASE_URL = "https://staging-api.realdevsquad.com";
+const BASE_URL = "http://localhost:3000";
```

Comment:
Base url would be `https://staging-api.realdevsquad.com` now

Reply:
Changed!